leftwm/doc/leftwm.1

```diff
@@ -0,0 +1,339 @@
+.IP
+With this file you can configure the modkey for leftwm, available keybinds, tag names, workspace layouts and more.
+.RS
+NOTE: This file is automatically generated when leftwm or leftwm-check is run for the first time.
+.SS Default keybinds
+.IP "Movement"
+ up, down, left or right refer to the arrow keys in your keyboard.
+.TS
+tab(@);
+l l.
+T{
+Keybinding
+T}@T{
+Description
+T}
+_
+T{
+Mod + (1-9)
+T}@T{
+Switch to a desktop/tag
+T}
+T{
+Mod + Shift + (1-9)
+T}@T{
+Move the focused window to desktop/tag
+T}
+T{
+Mod + W
+T}@T{
+Switch the desktops for each screen.
+Desktops [1][2] changes to [2][1]
+T}
+T{
+Mod + Shift + W
+T}@T{
+Move window to the other desktop
+T}
+T{
+Mod + (up or down)
+T}@T{
+Focus on the different windows in the current workspace
+T}
+T{
+Mod + Shift + (up or down)
+T}@T{
+Move the different windows in the current workspace
+T}
+T{
+Mod + Enter
+T}@T{
+Move selected window to the top of the stack in the current workspace
+T}
+T{
+Mod + Ctrl + (up or down)
+T}@T{
+Switch between different layouts
+T}
+T{
+Mod + Shift + (left or right)
+T}@T{
+Switch between different workspaces
+T}
+T{
+Mod + Shift + Enter
+T}@T{
+Open a terminal
+T}
+T{
+Mod + Ctrl + L
+T}@T{
+Lock the screen
+T}
+T{
+Mod + Shift + X
+T}@T{
+Exit LeftWM
+T}
+T{
+Mod + Shift + Q
+T}@T{
+Close the current window
+T}
+T{
+Mod + Shift + R
+T}@T{
+Reload LeftWM and its config
+T}
+T{
+Mod + p
+T}@T{
+Use dmenu to start application
+T}
+.TE
+.PP
+.B Note:
+Although we encourage you to use Alacritty, LeftWM will set your default terminal to the first terminal it finds in this list (in the order presented):
+.IP \[bu] 2
+Alacritty
+.IP \[bu] 2
+Termite
+.IP \[bu] 2
+Kitty
+.IP \[bu] 2
+URXVT
+.IP \[bu] 2
+RXVT
+.IP \[bu] 2
+ST
+.IP \[bu] 2
+ROXTerm
+.IP \[bu] 2
+Eterm
+.IP \[bu] 2
+XTerm
+.IP \[bu] 2
+Terminator
+.IP \[bu] 2
+Terminology
+.IP \[bu] 2
+Gnome Terminal
+.IP \[bu] 2
+XFCE4 Terminal
+.IP \[bu] 2
+Konsole
+.IP \[bu] 2
+UXTerm
+.IP \[bu] 2
+Guake
+
+.IP "About Keybind Modifiers"
+Note: All entries require a modifier, even if blank: modifier = []
+see your
+.I
+config.toml
+file for more information.
+
+.SS Floating Windows
+.PP
+You can optionally switch between tiling or floating mode for any
+window.
+.PP
+.TS
+tab(@);
+l l.
+T{
+Keybinding
+T}@T{
+Description
+T}
+_
+T{
+Mod + MouseDrag
+T}@T{
+Switch a tiled window to floating mode
+T}
+T{
+Mod + RightMouseDrag
+T}@T{
+Resize a window
+T}
+T{
+Drag window onto a tile
+T}@T{
+Switch a floating window to tiling mode
+T}
+.TE
+.SS FloatingToTile
+This behaviour snaps the currenly focused window into the workspace below.
+.SS TileToFloating
+This behaviour switches the currently focused window to floating mode when it's tiled.
+.SS ToggleFloating
+This behaviour switches the currenly focused window between floating and tiled mode.
+
+.\" Configuration section\"
+.SH CONFIGURATION
+.B IMPORTANT:
+You will need to You will need to reload in order to apply any changes you write to
```

Comment:
~You will need to~ You will need to

Reply:
I just realized I was recording a kbd macro in vim when writing (pressed q twice xD) that's why some text is duplicated.